src/language/pythonLanguage.ts

```diff
@@ -0,0 +1,77 @@
+import { buildParameterTypeLinksFromMatches } from './helpers.js'
+import { Language } from './types.js'
+
+export const pythonLanguage: Language = {
+  defineParameterTypeQueries: [
```

Comment:
Make this an empty array. Behave doesn't support Cucumber Expressions. (Parameter types is a Cucumber Expressions concept that doesn't work with Regular Expressions).